src/gerrit_argo_gateway/__main__.py

```diff
@@ -1,22 +1,27 @@
+"""
+Main module to be invoked via python -m gerrit_argo_gateway.
+
+Setups loggiging and the gateway and hooks up the signal handlers to stop the thread.
```

Comment:
typo: "loggiging" :D

Is "setups" the right word? I'd have chosen "sets up". I could not find "setup" as a verb in my online dictionary of choice.